pkg/apis/picchu/v1alpha1/releasemanager_types.go

```diff
@@ -93,6 +93,20 @@ type ReleaseManagerRevisionMetricsStatus struct {
 	DeploySeconds                 *float64 `json:"deploySeconds,omitempty"`
 	CanarySeconds                 *float64 `json:"canarySeconds,omitempty"`
 	ReleaseSeconds                *float64 `json:"releaseSeconds,omitempty"`
+
+	GitCreateSecondsInt              *int `json:"gitCreateSecondsInt,omitempty"`
```

Comment:
oof. this might be worth cleaning up after the conversion. Machu ends up turning these back into absolute timestamps

Reply:
Oof I missed that machu used these. I'll have to make some updates to this PR then, thanks!